packages/manager/modules/adp/src/deploy/review.html

```diff
@@ -0,0 +1,138 @@
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="admin_mail"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_cloud_project_email">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="admin_mail" data-ng-bind="$ctrl.adp.admin_mail"></span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- nodes config -->
+    <section class="mb-3">
+        <h6 data-translate="adp_deploy_review_nodes_title"></h6>
+        <div class="border-top">
+            <form>
+                <div class="form-group row" data-ng-repeat="(nodeType, node) in $ctrl.nodesConfig track by nodeType">
+                    <label for="{{ ::nodeType }}_nodes"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_nodes_{{ ::nodeType }}">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="{{ ::nodeType }}_nodes">
+                            {{ ::node.count }} x {{ node.selectedFlavor.displayName }}
+                        </span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- storage config -->
+    <section class="mb-3">
+        <h6 data-translate="adp_deploy_review_storage_title"></h6>
+        <div class="border-top">
+            <form>
+                <div class="form-group row">
+                    <label for="raw_storage"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_storage_raw">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="raw_storage" data-ng-bind="$ctrl.adp.hdfs_effective_storage"></span>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="usable_storage"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_storage_usable">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="usable_storage" data-ng-bind="$ctrl.adp.edge_node_storage"></span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- price details -->
+    <div class="mb-4">
+        <div class="row border rounded border-secondary bg-light">
```

Comment:
Is [`box`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/box) can fit your need?

Reply:
This is done